SocialCareCaseViewerApi/V1/Boundary/Requests/UpdateCaseStatusRequest.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+using FluentValidation;
+
+#nullable enable
+namespace SocialCareCaseViewerApi.V1.Boundary.Requests
+{
+    public class UpdateCaseStatusRequest
+    {
+        [JsonPropertyName("personId")]
+        public long PersonId { get; set; }
+
+        [JsonPropertyName("endDate")]
+        public DateTime? EndDate { get; set; }
+
+        [JsonPropertyName("editedBy")]
+        public string EditedBy { get; set; } = null!;
+
+        [JsonPropertyName("notes")]
+        public string? Notes { get; set; }
+
+        [JsonPropertyName("values")]
+        public List<CaseStatusValue>? Values { get; set; }
+    }
+
+    public class CaseStatusValue
+    {
+        [JsonPropertyName("name")]
+        public string Name { get; set; } = null!;
+
+        [JsonPropertyName("value")]
+        public string Value { get; set; } = null!;
+    }
+
+    public class UpdateCaseStatusValidator : AbstractValidator<UpdateCaseStatusRequest>
+    {
+        public UpdateCaseStatusValidator()
+        {
+            RuleFor(x => x.EndDate)
```

Comment:
I'm not sure but I think that `end date` could also be in the past but must be after the start date. @mapleleafjack Is this correct?

Reply:
That's right, `end date` can be either in the past or future, but not before the start date of the Case Status!

The UI already prevents to send a date past the Case Status start but having a check here will probably be useful :)